HISTORY.md

```diff
@@ -1144,26 +1180,68 @@
 
 ## 14.0.13 alpha 2020-02-19
 
+* fix(linux): setup gschema (#2675)
+* fix(ios): Responsive predictive toggles (#2674)
+* chore(windows): cleanup documentation (#2681)
 * chore: merge p9s2 beta part 2 to master (#2683)
 
 ## 14.0.12 alpha 2020-02-18
 * fix(android/samples): Fix build for KMSample1 project (#2669)
+* fix(developer): crash in context help (#2661)
+* fix(developer): crash importing blank OSK (#2663)
+* fix(common, linux): Misc keyboard processor fixes for Xenial (#2648)
```

Comment:
I'd like to specify the core scope. Otherwise `common` will show in all platform histories.

```suggestion
* fix(common/core/desktop, linux): Misc keyboard processor fixes for Xenial (#2648)
```